docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -42,8 +42,8 @@ Prometheus node_exporter is a widely used tool that exposes system metrics. Inst
         # metrics_path defaults to '/metrics'
         # scheme defaults to 'http'.
 
-        static_configs:
-        - targets: ['<hostname>:9100']
+          static_configs:
+          - targets: ['<hostname>:9100']
```

Comment:
```suggestion
               - targets: ['<hostname>:9100']
```

I think per the [Prometheus docs](https://github.com/prometheus/prometheus/blob/release-2.34/config/testdata/conf.good.yml#L51-L74), `targets` should be further indented as a child parameter of `static_configs`. This suggestion should apply this change with a 4-space indent consistent with the rest of the example.

Reply:
I have made the suggested changes. Please check and let me know if there is any other changes need to be made.